index.js

```diff
@@ -497,6 +500,7 @@ function init(aApp, aPredefinedSpec = {}, aSpecOutputPath = undefined, aWriteInt
     tags: aTags,
     ignoredNodeEnvironments: aIgnoredNodeEnvironments,
     alwaysServeDocs: aAlwaysServeDocs,
+    swaggerDocumentOptions: aSwaggerDocumentOptions,
```

Comment:
might as well re-order them here to keep it consistent.

Reply:
i put in the correct order too